### PR TITLE
[master] add support for setting auditConfig

### PR DESCRIPTION
**Pull request by detiber**
Adds support for https://github.com/openshift/origin/pull/8815

### CODE REVIEW

**Comment by detiber** [comment]
aos-ci-test

**Comment by detiber** [comment]
@abutcher @sdodson ptal, I'm currently testing.

@etsauer heads up

File: roles/openshift_master/templates/master.yaml.v1.j2 [hunk 1]
```diff
@@ -39,6 +39,9 @@ assetConfig:
     maxRequestsInFlight: 0
     requestTimeoutSeconds: 0
 {% if openshift_master_ha | bool %}
+{% if openshift.master.audit_config | default(none) is not none and openshift.common.version_gte_3_2_or_1_2 %}
```
Comment (abutcher) [inline roles/openshift_master/templates/master.yaml.v1.j2]: `openshift.common.version_gte_3_2_or_1_2 | bool`

Comment (detiber) [inline roles/openshift_master/templates/master.yaml.v1.j2]: Indeed, fixed.

**Comment by abutcher** [comment]
lgtm

**Comment by sdodson** [comment]
LGTM

**Comment by detiber** [comment]
pushed a fix to the examples to remove quoting around the boolean, otherwise local tests are showing this is good to go.

**Comment by detiber** [comment]
aos-ci-test

**Comment by detiber** [comment]
@sdodson this should be good to go. We'll need to backport to 3.2 as well, adding that label now.

**Pull request merged**